app/src/main/java/com/example/sharewhatyoucanproject/login/LoginViewModel.kt

```diff
@@ -123,12 +123,12 @@ class LoginViewModel(
 }
 
 class LoginViewModelFactory(
-    private val app: Application,
+    private val app: Application?,
```

Comment:
why is this nullable? If you don't need the context then you can remove the application

Reply:
removed